src/shared/use-data-value-set/query-key-factory.js

```diff
@@ -5,14 +5,14 @@ export const dataValueSets = {
         periodId,
         orgUnitId,
         categoryComboId,
-        categoryOptionIds = [],
+        categoryOptionIds,
     }) => {
         const params = {
             ds: dataSetId,
             pe: periodId,
             ou: orgUnitId,
             cc: categoryComboId,
-            cp: categoryOptionIds.join(';'),
+            cp: categoryOptionIds && categoryOptionIds.join(';'),
```

Comment:
this change is to have cp be undefined rather than an empty string; the empty string would be included in the payload and cause errors; it seemed easier to adjust here rather than adding logic in generating the payload in use-set-form-completion-mutation.js